js/data.json

```diff
@@ -604,13 +604,79 @@ openings = [
+		"name" : "Jojo's Bizarre Adventure : Battle Tendency OP - BLOODY STREAM",
+		"img" : "https://i.ytimg.com/vi/RZhGITHIbns/maxresdefault.jpg"
+	},
+
+	{
+		"link" : "https://cs1.mp3.pm/download/62867059/QzU4UnVIZTVSU01DQVc3eDRxeTlSYlphMHFxY21mZytJeDVLeDRZYUFYa3dtWHkwTW5zSFRjd04rbks4aGFxc1Z0dEt3QjlYUzRTZjQ3ZHVkektWMUh5KytWRTZOTnRBSjRMZ20xZktkNC81MTNvSkVzMVN3bWk0V2NvOEtRdXQ/JoJo_s_Bizarre_Adventure_Part_3_-_Stand_Proud_(mp3.pm).mp3",
+		"name" : "Jojo's Bizarre Adventure : Stardust Crusaders OP 1 - STAND PROUD",
+		"img" : "https://i.ytimg.com/vi/lviyLEF7lPE/maxresdefault.jpg"
+	},
+
+	{
+		"link" : "https://cs1.mp3.pm/download/159731123/QzU4UnVIZTVSU01DQVc3eDRxeTlSYlphMHFxY21mZytJeDVLeDRZYUFYblFWbXMzTWUybm5SQlQ3VXpBa055ZDdzSkJKRUh0M3JqKzhzWDkwTW8xaGVnMUFpU01na3hObitnYjl2ZllycS9GUVBDYVpjVFhEdVcxTHphQ2NGeS8/JOSTARS_-_JoJo_Sono_Chi_no_Kioku_end_of_THE_WORLD_(mp3.pm).mp3",
+		"name" : "Jojo's Bizarre Adventure : Stardust Crusaders OP 2 - Sono Chi No Kioku ~end Of THE WORLD~",
+		"img" : "https://i.ytimg.com/vi/9xF5GfqJrhE/maxresdefault.jpg"
+	},
+
+	{
+		"link" : "https://cs1.mp3.pm/download/87166424/QzU4UnVIZTVSU01DQVc3eDRxeTlSYlphMHFxY21mZytJeDVLeDRZYUFYazAyZUlpU1R4cUNwWktpb1A4WnFnWkcwMi81dE1VbzBzdmsxUE9LbUFBWSs1dkdERnhBZytzNXlKS1pvcjMwNm5aZ0lndzlwSkZZc1kyTjhEVG45bEk/THE_DU_-_Crazy_Noisy_Bizarre_Town_JoJo_s_Bizarre_Adventure_Opening_5_(mp3.pm).mp3",
+		"name" : "Jojo's Bizarre Adventure : Diamond Is Unbreakable OP 1 - Crazy Noisy Bizarre Town",
+		"img" : "https://i.ytimg.com/vi/Oz4TQ_ukGOM/maxresdefault.jpg"
+	},
+
+	{
+		"link" : "https://cs1.mp3.pm/download/99218017/QzU4UnVIZTVSU01DQVc3eDRxeTlSYlphMHFxY21mZytJeDVLeDRZYUFYbml2VTRGSkc4N1RTOS9uTmlMd3lXdnBkZHlxT0c2SFFLYWVQOUk5WEE0bjF2UFE4TEtKL0h0N0RpT1pqQlEvUzJkYU8wWVZFWDJXLzdZbzVuSmVNajM/THE_DU_-_Crazy_Noisy_Bizarre_Town_EDM_Arrange_Ver._(mp3.pm).mp3",
+		"name" : "Jojo's Bizarre Adventure : Diamond Is Unbreakable OP 1.1 - Crazy Noisy Bizarre Town (EDM Version)",
+		"img" : "https://s2.dmcdn.net/v/Gg-bP1NdCNTSBL7sj/x1080"
+	},
+
+	{
+		"link" : "https://cs1.mp3.pm/download/87166423/QzU4UnVIZTVSU01DQVc3eDRxeTlSYlphMHFxY21mZytJeDVLeDRZYUFYazAyZUlpU1R4cUNwWktpb1A4WnFnWkZqTWlReU53L2ZLV0U3STBRQ1Y3MGRTTWZYN2IzK2JKcFRvSzZPSUVLdXNFazh1dVI3aDRtNkFVZHQyTDFuN3c/batta_-_chase_JoJo_s_Bizarre_Adventure_Opening_6_(mp3.pm).mp3",
+		"name" : "Jojo's Bizarre Adventure : Diamond Is Unbreakable OP 2 - chase",
+		"img" : "https://animekaizoku.com/wp-content/uploads/2019/03/Jojo_keyart.jpg"
+	},
+
+	{
+		"link" : "https://cs1.mp3.pm/download/82680515/QzU4UnVIZTVSU01DQVc3eDRxeTlSYlphMHFxY21mZytJeDVLeDRZYUFYbFk4WDFlbFljQTRDMVZSL09CRFFOK2lVQTdrZnJBaXdSUWxZczhKcWpqczVIZXVVOElXT2xLOHcvdFBwSlB5aksySHhJelNiVUNXMWg2UDltdGh5SFM/JoJo_no_Kimyou_na_Bouken_Diamond_wa_Kudakenai_OP_3_-_Great_Days_-_by_Karen_Aoki_Daisuke_Hasegawa_(mp3.pm).mp3",
+		"name" : "Jojo's Bizarre Adventure : Diamond Is Unbreakable OP 3 - Great Days",
+		"img" : "https://i.ytimg.com/vi/wyhBSrfMEnw/maxresdefault.jpg"
```

Comment:
Hi, @the-bose  there thank you for your contribution. Links seems to be working fine.
Just a small change. Can you replace this image as it has some text at the bottom and it would look good on the background.